docs/products/kafka/kafka-connect/howto/kafka-connect-debezium-tls-pg.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Integrate Apache Kafka Connect with PostgreSQL using Debezium and mutual TLS
+---
+
+Learn how to set up Apache Kafka Connect with PostgreSQL and Debezium using mutual TLS for secure data synchronization.
+
+## Overview
+
+Integrating Apache Kafka Connect with PostgreSQL using Debezium and mutual TLS (mTLS)
+enhances data security by requiring the client and server to authenticate each other.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+
+- Access to an Aiven for Apache Kafka service with
+[Apache Kafka Connect](/docs/products/kafka/kafka-connect/howto/enable-connect)
+enabled.
+- Administrative access to a PostgreSQL database with SSL enabled.
+- The following SSL certificates and keys obtained from your PostgreSQL database:
+  - SSL client certificate: The public certificate for client authentication.
+  - SSL root certificate: The certificate of the Certificate Authority (CA) that signed
+  the servers' and clients' certificates.
+  - SSL client key: The client's private key is used to encrypt the data sent to the
+  server.
+  For additional details, see
+  [Certificate requirements](/docs/platform/concepts/tls-ssl-certificates#certificate-requirements).
```

Comment:
Maybe we can also specify in case of cloud sql which are the steps to:
1. unlock the aiven ips (that are required to be whitelisted otherwise they get an error because the connection itsn't able to reach the db)
2. enable the 2 features I've mentioned initially in the presentation, the one for using the right level for the read only log (wal_level) and the other one for reading the log and being able to serialize/deserialize it from/into json. There are two different plugins I've mentioned only one in the slides if I'm not remembering badly they are `wal2json` and `pgoutput`

Reply:
wal2json is about to be deprecated with the new DBZ 2.5 rollout so we should mention pgoutput and perhaps decoderbufs which are identified as the wal2json replacements in the docs

Reply:
Add a subsection for CloudSQL and a note about wal2json deprecation.